Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -379,17 +379,21 @@ static bool isRubyContainerOrInternalRubyBox(const RenderStyle& style)
 }
 
 // https://drafts.csswg.org/css-ruby-1/#bidi
-static UnicodeBidi adjustUnicodeBidiForRuby(UnicodeBidi unicodeBidi)
+static UnicodeBidi forceBidiIsolationForRuby(UnicodeBidi unicodeBidi)
 {
     switch (unicodeBidi) {
     case UnicodeBidi::Normal:
     case UnicodeBidi::Embed:
+    case UnicodeBidi::Isolate:
         return UnicodeBidi::Isolate;
     case UnicodeBidi::Override:
+    case UnicodeBidi::IsolateOverride:
         return UnicodeBidi::IsolateOverride;
-    default:
-        return unicodeBidi;
+    case UnicodeBidi::Plaintext:
+        return UnicodeBidi::Plaintext;
     }
+    ASSERT_NOT_REACHED();
+    return UnicodeBidi::Isolate;
```

Comment:
I actually think the old code read more similarly to the spec. No strong opinion here though.

> Bidi isolation is forced on all [internal ruby boxes](https://drafts.csswg.org/css-ruby-1/#internal-ruby-boxes) and the [ruby container](https://drafts.csswg.org/css-ruby-1/#ruby-container): the [normal](https://drafts.csswg.org/css-writing-modes-4/#valdef-unicode-bidi-normal) and [embed](https://drafts.csswg.org/css-writing-modes-4/#valdef-unicode-bidi-embed) values of [unicode-bidi](https://drafts.csswg.org/css-writing-modes-3/#propdef-unicode-bidi) compute to [isolate](https://drafts.csswg.org/css-writing-modes-4/#valdef-unicode-bidi-isolate), and [bidi-override](https://drafts.csswg.org/css-writing-modes-4/#valdef-unicode-bidi-bidi-override) computes to [isolate-override](https://drafts.csswg.org/css-writing-modes-4/#valdef-unicode-bidi-isolate-override).